rtil/src/native/mod.rs

```diff
@@ -44,16 +44,16 @@ use crate::semaphore::Semaphore;
 #[cfg(windows)] use self::windows::*;
 
 #[cfg(unix)] pub use self::linux::INITIALIZE_CTOR;
-#[cfg(windows)] pub use self::windows::{DllMain, suspend_threads, resume_threads};
+#[cfg(windows)] pub use self::windows::{resume_threads, suspend_threads, DllMain};
 pub use self::character::AMyCharacter;
 pub use self::slateapp::{
-    FSlateApplication,
-    unhook_fslateapplication_onkeydown,
     hook_fslateapplication_onkeydown,
-    unhook_fslateapplication_onkeyup,
     hook_fslateapplication_onkeyup,
-    unhook_fslateapplication_onrawmousemove,
     hook_fslateapplication_onrawmousemove,
+    unhook_fslateapplication_onkeydown,
+    unhook_fslateapplication_onkeyup,
+    unhook_fslateapplication_onrawmousemove,
```

Comment:
These should stay grouped by function rather than alphabetically sorted. This is one of the few cases where an `#[allow(…)]` attribute makes sense.